migrations/migrations/0003_motion_state_dont_set_identifier.py

```diff
@@ -0,0 +1,14 @@
+from datastore.migrations import AddFieldMigration
+
+
+class Migration(AddFieldMigration):
+    """
+    This migration adds `motion_state/dont_set_identifier` with a
+    default of False to each motion_state.
+    """
+
+    target_migration_index = 4
+
+    collection = "motion_state"
+    field = "dont_set_identifier"
+    default = False
```

Comment:
Migration would not have been necessary here, since the default is `False`, but it doesn't hurt